### PR TITLE
replace codacy with codeconv

### PR DESCRIPTION
**Pull request by appinteractive**
use https://codecov.io for code coverage reports on pullrequests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,12 +33,12 @@
...
   },
   "scripts": {
-    "ci": "yarn run eslint && yarn run test:coverage",
+    "ci": "yarn run eslint && yarn run test && yarn run test:coverage",
     "clean": "rm -Rf tmp",
     "mongo": "mongod --dbpath data --quiet",
     "mongo:win": "mongod --dbpath /data/db --quiet",
-    "test": "yarn run mocha && yarn run cucumber",
-    "test:coverage": "nyc --reporter=lcovonly --reporter=text-summary yarn run test",
+    "test": "nyc --reporter=text-lcov yarn run mocha && yarn run cucumber",
```
Comment (roschaefer) [inline package.json:40]: Does `nyc` consider the coverage by both mocha and cucumber? Do we want to have code coverage for `test` at all?

File: .codecov.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+coverage:
+  range: "60...100"
```
Comment (roschaefer) [inline .codecov.yml:2]: Minimum code coverage 60%?

**Review by roschaefer — APPROVED** [review]
I don't know exactly what the code coverage tool is doing. See the comments in my review. @appinteractive if you can confirm my assumptions, feel free to merge (or change and merge respectively).

File: .codecov.yml (hunk 2 above)
Comment (appinteractive) [reply .codecov.yml:2]: As our current coverage is 68% I did not want to lock the current pullrequests when leaving the standard on 70%

File: package.json (hunk 1 above)
Comment (appinteractive) [reply package.json:40]: Im not sure to be honest. I would say we have to keep a look on it.

**Pull request merged**